backend/cellguide/pipeline/canonical_marker_genes/canonical_markers.py

```diff
@@ -34,7 +34,10 @@ def __init__(self, *, wmg_tissues: list[str], wmg_human_genes: list[str]):
         """
 
         logger.info("Fetching ASCTB data...")
-        self.asctb_data = requests.get(ASCTB_MASTER_SHEET_URL).json()
+        session = setup_retry_session()
```

Comment:
setup_retry_session is more robust than requests.get because it retries if connection is interrupted.